wagtail/admin/views/account.py

```diff
@@ -138,10 +138,18 @@ def render(self):
 
 class NameEmailSettingsPanel(BaseSettingsPanel):
     name = "name_email"
-    title = gettext_lazy("Name and Email")
+    # title = gettext_lazy("Name and Email")
     order = 100
     form_class = NameEmailForm
 
+    @cached_property
```

Comment:
Please be sure to run the code Locally and check when making changes, this line is now failing as this is not imported. 

Do some searching in other code to see how this is used/imported.
Thank you.

Reply:
sorry! 
I'll make it correct.

Reply:
sorry!
I'll fix it right away.